### PR TITLE
Move more commands from BaseLayout to BaseLayoutCommandsViewModel

### PR DESCRIPTION
**Pull request by d2dyno1**
**Resolved / Related Issues**
This PR is an ongoing work to improve Files' codebase and implement MVVM pattern. For #4180 

**Details of Changes**
This PR moves some functions out of `BaseLayout` and implements them in `BaseLayoutCommandsViewModel`

**Validation**

How did you test these changes?
- [x] Built and ran the app
- [x] Tested the changes for accessibility

### CODE REVIEW

**Comment by d2dyno1** [comment]
Ready!

File: Files/Views/LayoutModes/GenericFileBrowser.xaml [hunk 1]
```diff
@@ -684,8 +684,12 @@
                     x:Name="PinItemToStart"
                     x:Uid="PinItemToStart"
                     x:Load="False"
-                    Click="PinItemToStart_Click"
                     Text="Pin to the Start Menu">
```
Comment (yaichenbaum) [inline Files/Views/LayoutModes/GenericFileBrowser.xaml:687]: Can't you use the command event?

Comment (d2dyno1) [reply Files/Views/LayoutModes/GenericFileBrowser.xaml:687]: To pass in `EventArgs` you need to do it the way it's done now.

Comment (yaichenbaum) [reply Files/Views/LayoutModes/GenericFileBrowser.xaml:687]: @d2dyno1 Would setting the `CommandParameter` help?

Comment (d2dyno1) [reply Files/Views/LayoutModes/GenericFileBrowser.xaml:687]: nope

**Review by yaichenbaum — APPROVED** [review]

**Comment by yaichenbaum** [comment]
@d2dyno1 Great work!

File: Files/Helpers/ContextFlyoutItemHelper.cs [hunk 2]
```diff
@@ -486,7 +486,7 @@ public static List<ContextMenuFlyoutItemViewModel> GetBaseItemMenuItems(BaseLayo
                 {
                     Text = "PinItemToStart2".GetLocalized(),
```
Comment (yaichenbaum) [inline Files/Helpers/ContextFlyoutItemHelper.cs:487]:
```suggestion
                    Text = "PinItemToStart/Text".GetLocalized(),
```

File: Files/Helpers/ContextFlyoutItemHelper.cs [hunk 3]
```diff
@@ -495,7 +495,7 @@ public static List<ContextMenuFlyoutItemViewModel> GetBaseItemMenuItems(BaseLayo
                 {
                     Text = "UnpinItemFromStart2".GetLocalized(),
```
Comment (yaichenbaum) [inline Files/Helpers/ContextFlyoutItemHelper.cs:496]:
```suggestion
                    Text = "UnpinItemFromStart/Text".GetLocalized(),
```

These strings were renamed in `main`.

**Review by yaichenbaum — APPROVED** [review]

**Pull request merged**